cloudbuild.yaml

```diff
@@ -30,22 +30,29 @@ steps:
     waitFor: ['build_image']
     timeout: 5m
     args: ['./scripts/git_init']
+
   - name: 'gcr.io/oak-ci/oak:latest'
-    id: check_formatting
+    id: bazel_init
+    entrypoint: 'bazel'
     waitFor: ['build_image']
+    timeout: 5m
+    args: ['info']
```

Comment:
`info` is used just to unpack Bazel, right?

Reply:
Yes, clarified.